openvswitch: compilation error

Maintainer: @stintel  @paulfertser @dhewg 
Environment: mediatek_filogic , Asus TUF AX4200, main commit 560965d5826626e3be8c1f1db194db43cc7002cf

Description:
Hi, 
I tried to build openwrt from branch main ( commit 560965d5826626e3be8c1f1db194db43cc7002cf - Thu Aug 31 ).
I got errors like below for package openvswitch.

```
touch /usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/.built_check
CFLAGS="-Os -pipe -mcpu=cortex-a53 -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0=openvswitch-2.17.0 -flto=auto -fno-fat-lto-objects -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -std=gnu99  -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/usr/include -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/include/fortify -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/include " CXXFLAGS="-Os -pipe -mcpu=cortex-a53 -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0=openvswitch-2.17.0 -flto=auto -fno-fat-lto-objects -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -std=gnu99 -flto=auto -fno-fat-lto-objects  -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/usr/include -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/include/fortify -I/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/include " LDFLAGS="-L/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/usr/lib -L/usr/build/builds/openwrt/staging_dir/toolchain-aarch64_cortex-a53_gcc-12.3.0_musl/lib -fuse-ld=bfd -flto=auto -fuse-linker-plugin -znow -zrelro " PYTHONPATH="/usr/build/builds/openwrt/staging_dir/hostpkg/lib/python3.11:/usr/build/builds/openwrt/staging_dir/hostpkg/lib/python3.11/site-packages" make  -C /usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/. AR="aarch64-openwrt-linux-musl-gcc-ar" AS="aarch64-openwrt-linux-musl-gcc -c -Os -pipe -mcpu=cortex-a53 -fno-caller-saves -fno-plt -fhonour-copts -ffile-prefix-map=/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0=openvswitch-2.17.0 -flto=auto -fno-fat-lto-objects -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -std=gnu99" LD="aarch64-openwrt-linux-musl-ld.bfd" NM="aarch64-openwrt-linux-musl-gcc-nm" CC="aarch64-openwrt-linux-musl-gcc" GCC="aarch64-openwrt-linux-musl-gcc" CXX="aarch64-openwrt-linux-musl-g++" RANLIB="aarch64-openwrt-linux-musl-gcc-ranlib" STRIP=aarch64-openwrt-linux-musl-strip OBJCOPY=aarch64-openwrt-linux-musl-objcopy OBJDUMP=aarch64-openwrt-linux-musl-objdump SIZE=aarch64-openwrt-linux-musl-size CROSS="aarch64-openwrt-linux-musl-" ARCH="aarch64" ;
make[4]: Entering directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
make  all-recursive
make[5]: Entering directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
Making all in datapath
make[6]: Entering directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/datapath'
make[7]: Entering directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/datapath'
make[7]: Leaving directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/datapath'
make[6]: Leaving directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/datapath'
make[6]: Entering directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
an.tmac:lib/ovs-fields.7:762: warning: tbl preprocessor failed, or it or soelim was not run; table(s) likely not rendered (TE macro called with TW register undefined)
troff:vswitchd/ovs-vswitchd.8:1290: warning: cannot select font 'CW'
make[6]: *** [Makefile:5684: manpage-check] Error 1
make[6]: Leaving directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
make[5]: *** [Makefile:4431: all-recursive] Error 1
make[5]: Leaving directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
make[4]: *** [Makefile:2333: all] Error 2
make[4]: Leaving directory '/usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0'
make[3]: *** [Makefile:280: /usr/build/builds/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/openvswitch-2.17.0/.built] Error 2
make[3]: Leaving directory '/usr/build/builds/openwrt/feeds/packages/net/openvswitch'
time: package/feeds/packages/openvswitch/compile#1.44#0.37#1.68
    ERROR: package/feeds/packages/openvswitch failed to build.
make[2]: *** [package/Makefile:120: package/feeds/packages/openvswitch/compile] Error 1

```
UPDATE1: As a workarond we can downgrade  groff from version 1.23.0 to 1.22.4. Error occurs only with groff version 1.23.


best regards,
Patryk

--- a/net/openvswitch/Makefile
+++ b/net/openvswitch/Makefile
@@ -251,6 +251,7 @@ CONFIGURE_ARGS+= \
 CONFIGURE_VARS += \
 	$(if $(CONFIG_OPENVSWITCH_WITH_LIBUNBOUND),,ac_cv_lib_unbound_ub_ctx_create=no) \
 	ovs_cv_flake8=no \
+	ovs_cv_groff=no \
 	ovs_cv_python3=$(PYTHON3) \
 	ovs_cv_python3_host=$(HOST_PYTHON3_BIN) \
 	SPHINXBUILD=none \

